apps/hmc/hmc-hmi-inbound-adapter/perftest.yaml

```diff
@@ -5,4 +5,11 @@ metadata:
 spec:
   values:
     java:
-      memoryRequests: '1Gi'
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 4
+      memoryRequests: '1536Mi'
+      memoryLimits: '2048Mi'
+      cpuRequests: '100m'
```

Comment:
You're proposing decreasing CPU from 250/1500 to 100/250. Whilst this seems like a more suitable config based on the current CPU utilisation in Prod, we found in the recent F&P HPA that reducing the CPU this much may starve the application on startup, leading to errors or very slow startup times (we saw both in F&P). I suggest leaving this at 250/1500 for now, until this is tested at lower levels.